### ISSUE TITLE
Node version

### ISSUE THREAD

**Issue by lamphamsy** [scality/eclib#42]
Should we remove tests for old node versions such as 0.10, 0.12?

**Comment by ghost** [comment]
I think we could, as we have no need to be backwards compatible for this library.

**Comment by ghost** [comment]
There are people using this library if I trust https://www.npmjs.com/package/eclib so we'll need a major version bump. That would be acceptable if we're bundling that with every other planned breaking change.

**Comment by lamphamsy** [comment]
I am not sure that that would be a breaking change. Node 0.10 and 0.12 are too old now.

**Comment by ghost** [comment]
Yeah, npm stats aren't explicitly stating which node versions are being used. (And not supporting old versions anymore is a breaking change, there's no way around that fact)

**Comment by ghost** [comment]
To be clear, I am in favor of updating the library to recent standards, but this should not be done too hastily.
